microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,24 +9,112 @@ ms.prod: microsoft-edge
+The User Agent \(UA\) can be used detect what version of a specific browser is being used on a certain operating system. Microsoft Edge offers two mechanisms for retrieving User Agent information: via User Agent Client Hints (recommended) and the legacy User Agent string. These can be accessed as follows: 
+
+**User Agent Client Hints (Recommended)**
+-	*Server-side*: UA Client Hints are sent by default with the `Sec-CH-UA` `HTTPS` header 
+-	*Client-side*: UA Client Hints can be accessed via JavaScript by querying the value of `navigator.userAgentData`
+
+**User Agent String**
+-	*Server-side*: UA strings are sent by default with the User-Agent `HTTP` header 
+-	*Client-side*: UA strings can be accessed via JavaScript by querying the value of `navigator.userAgent`
 
 Microsoft recommends that web developers make use of [feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) whenever possible to improve code maintainability, reduce code fragility, and eliminate the risk of code breakage in the event of future UA string updates.  
 
 For cases where feature detection is not applicable and UA detection must be used, the format of the Microsoft Edge UA on desktop is as follows:
 
+## User Agent Client Hints
+Starting from Edge90 Stable, developers can access browser information in a cleaner, more privacy preserving way. 
+
+### UA-CH HTTPS Header
+
+By default, Chromium browsers like Edge will send the `Accept-CH-UA` request header in the following format: 
+
+```HTTP
+Sec-CH-UA: “Chromium”;v=”91”, “Microsoft Edge”;v=”91”,”Dummy;Browser Brand”;v=”99”
+Sec-CH-UA-Mobile: ?0
+```
+Note: Client Hints are only sent over `HTTPS`. 
+
+More information, if needed, can be accessed by sending subsequent requests with the desired headers: 
+
+#### User-Agent Request & Response Headers
+| Response Header | Example Response Value |
+| ----------- | ----------- |
+|`Sec-CH-UA`|`“Chromium”;v=”91”, “Microsoft Edge”;v=”91”,”GREASE”;v=”99"`|
+|`Sec-CH-UA-Mobile`|`false`|
+|`Sec-CH-UA-Full-Version`|`91.0.866.0`|
+|`Sec-CH-UA-Platform`|`Windows`|
+|`Sec-CH-UA Platform-Version`|`10.0`|
+|`Sec-CH-UA-Arch`|`x86`|
+|`Sec-CH-UA-Model`||
+
+### UA-CH JavaScript API
+
+The default return value from `navigator.userAgentData` is in the following format: 
+
+```javascript
+{brands: 
+    [{brand: “Chromium”,”version”:”91”}, 
+    brand: “Microsoft Edge”,”version”:”91”}, 
+    brand: “GREASE”,”version”:”99”},]
+mobile: false
+}
+```
+More detailed information, if needed, can be accessed with [getHighEntropyValues()](https://wicg.github.io/ua-client-hints/#getHighEntropyValues), for example, calling

```

Comment:
Delete ", if needed,".
Change:
, for example, calling ... May
to:
.  For example, calling ... may

Reply:
Updated.